tests/upgrade_cli_test.go

```diff
@@ -71,12 +70,6 @@ var _ = Describe("k3s upgrade manual test", Label("upgrade-with-cli"), func() {
 				return err
 			}, 10*time.Minute, 10*time.Second).ShouldNot(HaveOccurred())
 
-			var v string
-			Eventually(func() string {
-				v, _ = vm.Sudo(getVersionCmd)
-				return v
-				// TODO: Add regex semver check here
-			}, 10*time.Minute, 10*time.Second).Should(ContainSubstring("v"), v)
```

Comment:
This check seemed intentional. What changed?

Reply:
freeform semver for now.

Still this check is useless. Checking that the version just contains a v its absurd? Even if the upgrade didnt pass, this check would pass so I dont see how this is a useful check at all.

Reply:
I think we added that because we had a bug with a missing `v` . This covers it enough :D.